src/main/java/frc/robot/generated/TunerConstants.java

```diff
@@ -14,6 +14,8 @@
 import frc.robot.Constants;
 
 public class TunerConstants {
+    // constants for pivotMotor for shooter
+    public static final double pivotMotorAngleErrorThreashhold = 1.0 / 360.0;

```

Comment:
`TunerConstants` are technically for swerve. I'd ideally like to keep it just swerve constants and maybe rename this class if it's confusing. I also don't think all constants need to be in this one location. There's a lot of pros and cons for either approach of putting everything in one constants file vs putting them in the class they relate to. I tend to lean towards the latter approach.